tests/basic/storage_pool_test.go

```diff
@@ -3416,10 +3309,8 @@ var _ = Describe("{PoolMaintenanceModeResize}", func() {
 		log.FailOnError(err, fmt.Sprintf("Driver is down on node %s", stNode.Name))
 		dash.VerifyFatal(err == nil, true, fmt.Sprintf("PX is up after maintenance cycle on node %s", stNode.Name))
 		status, err = Inst().V.GetNodeStatus(*stNode)
+		log.FailOnError(err, "error get node [%s] status", stNode.Name)
```

Comment:
nit: Can we stick to one style of error for same thing, see line 3401 where you do `log.FailOnError(err, "failed to get node [%s] status", stNode.Name)` and here you do `log.FailOnError(err, "error get node [%s] status", stNode.Name)`, it will be easier to search through logs for these errors when need to. Could you please fix it everywhere, I saw it in multiple places, pick one style and stick to it.

Reply:
fixed